### ISSUE TITLE
[Bug]: ControlNet OpenPose model lost the personalized information of my dreambooth finetuned SD model

### ISSUE THREAD

**Issue by xiyichen** [Mikubill/sd-webui-controlnet#1935]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

I would like to create personalized pose-dependent images of Monica Geller from Friends with the ControlNet OpenPose model and a dreambooth finetuned SD 1.5 model. However, the images generated with ControlNet and my finetuned SD model doesn't look like the person that the model is finetuned with.

The image generated with the webui **without** ControlNet enabled, which looks good:
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/50854238/f73fb81d-15fa-40a3-822f-584bb4a0bf11)

The image generated with the webui **with** ControlNet enabled, doesn't look like Monica at all (although the pose is right):
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/50854238/d296c036-cbed-4aaa-9828-5dd423723166)

### Steps to reproduce the problem

1. Finetune dreambooth (using [this script](https://github.com/huggingface/diffusers/blob/main/examples/dreambooth/train_dreambooth.py)) with 5 input images using diffusers library (with prior-preservation loss and train_text_encoder turned on)
Images I used (Monica Geller from Friends): https://drive.google.com/drive/folders/1hngFxpPFnqo-VTxqoHhxB3aG-xfssZAJ?usp=sharing
Script to finetune with dreambooth:
```
export MODEL_NAME="stablediffusionapi/cyberrealistic"
export INSTANCE_DIR="./monica_dreambooth_all_masked_2x/"
export CLASS_DIR="./monica_class_dir"
export OUTPUT_DIR="./monica_masked_dreambooth_2x_pp_cr_checkpoints_800"

accelerate launch train_dreambooth.py \
  --pretrained_model_name_or_path=$MODEL_NAME  \
  --instance_data_dir=$INSTANCE_DIR \
  --class_data_dir=$CLASS_DIR \
  --output_dir=$OUTPUT_DIR \
  --train_text_encoder \
  --with_prior_preservation --prior_loss_weight=1.0 \
  --instance_prompt="a photo of sks woman" \
  --class_prompt="a photo of woman" \
  --resolution=512 \
  --train_batch_size=1 \
  --gradient_accumulation_steps=1 \
  --learning_rate=5e-6 \
  --lr_scheduler="constant" \
  --lr_warmup_steps=0 \
  --num_class_images=200 \
  --max_train_steps=800 \
  --report_to="wandb" \
  --validation_prompt="a photo of sks woman" \
  --validation_steps=100 \
  --seed="0"
```
2. Convert finetuned SD model to .safetensors for the webui to load (using [this script](https://github.com/huggingface/diffusers/blob/main/scripts/convert_diffusers_to_original_stable_diffusion.py))
3. open up webui, load the converted safetensors
4. enable an OpenPose ControlNet module, and generate with prompt "A full body photo of sks woman, best quality" and negative prompt "blurry, upper body only, naked, extra limbs, extra legs, more than 2 legs, more than 1 person, ugly, bad, unrealistic, cartoon, anime"

### What should have happened?

![smplx-a-pose_openpose_body_hand](https://github.com/Mikubill/sd-webui-controlnet/assets/50854238/755eb8f2-81cc-4ebb-8ba7-652d0bb887fb)
Given this "A pose", I would expect a generated image of Monica Geller in the "A pose".

### Commit where the problem happens

webui: 68f336bd994bed5442ad95bad6b6ad5564a5409a
controlnet: 0cfc88b6a892076d199e68c02e9c306ac6ab2ead

### What browsers do you use to access the UI ?

Google Chrome

### Command Line Arguments

```Shell
No
```

### List of enabled extensions

ControlNet

### Console logs

```Shell
################################################################
Install script for stable-diffusion + Web UI
Tested on Debian 11 (Bullseye)
################################################################

################################################################
Running on root user
################################################################

################################################################
Repo already cloned, using it as install directory
################################################################

################################################################
Create and activate python venv
################################################################

################################################################
Launching launch.py...
################################################################
Cannot locate TCMalloc (improves CPU memory usage)
Python 3.10.6 (main, Oct 24 2022, 16:07:47) [GCC 11.2.0]
Version: v1.5.1
Commit hash: 68f336bd994bed5442ad95bad6b6ad5564a5409a

Launching Web UI with arguments: -f
no module 'xformers'. Processing without...
no module 'xformers'. Processing without...
No module 'xformers'. Proceeding without it.
2023-08-20 06:29:45,337 - ControlNet - INFO - ControlNet v1.1.238
ControlNet preprocessor location: /root/stable-diffusion-webui/extensions/sd-webui-controlnet/annotator/downloads
2023-08-20 06:29:45,398 - ControlNet - INFO - ControlNet v1.1.238
Loading weights [10f0ff3f09] from /root/stable-diffusion-webui/models/Stable-diffusion/monica.safetensors
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 5.8s (launcher: 1.3s, import torch: 1.7s, import gradio: 0.7s, setup paths: 0.5s, other imports: 0.4s, load scripts: 0.6s, create ui: 0.4s, gradio launch: 0.1s).
Creating model from config: /root/stable-diffusion-webui/configs/v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Applying attention optimization: Doggettx... done.
Model loaded in 2.3s (load weights from disk: 0.7s, create model: 0.3s, apply weights to model: 0.5s, apply half(): 0.4s, move model to device: 0.3s).
2023-08-20 06:32:48,110 - ControlNet - INFO - Preview Resolution = 768
Downloading: "https://huggingface.co/lllyasviel/Annotators/resolve/main/body_pose_model.pth" to /root/stable-diffusion-webui/extensions/sd-webui-controlnet/annotator/downloads/openpose/body_pose_model.pth

100%|█████████████████████████████████████████████████████████| 200M/200M [00:05<00:00, 35.6MB/s]
Downloading: "https://huggingface.co/lllyasviel/Annotators/resolve/main/hand_pose_model.pth" to /root/stable-diffusion-webui/extensions/sd-webui-controlnet/annotator/downloads/openpose/hand_pose_model.pth

100%|█████████████████████████████████████████████████████████| 141M/141M [00:04<00:00, 30.9MB/s]
Downloading: "https://huggingface.co/lllyasviel/Annotators/resolve/main/facenet.pth" to /root/stable-diffusion-webui/extensions/sd-webui-controlnet/annotator/downloads/openpose/facenet.pth

100%|█████████████████████████████████████████████████████████| 147M/147M [00:03<00:00, 41.0MB/s]
2023-08-20 06:33:14,730 - ControlNet - INFO - Loading model: control_v11p_sd15_openpose [cab727d4]
2023-08-20 06:33:15,142 - ControlNet - INFO - Loaded state_dict from [/root/stable-diffusion-webui/extensions/sd-webui-controlnet/models/control_v11p_sd15_openpose.pth]
2023-08-20 06:33:15,142 - ControlNet - INFO - Loading config: /root/stable-diffusion-webui/extensions/sd-webui-controlnet/models/control_v11p_sd15_openpose.yaml
2023-08-20 06:33:17,763 - ControlNet - INFO - ControlNet model control_v11p_sd15_openpose [cab727d4] loaded.
2023-08-20 06:33:17,788 - ControlNet - INFO - Loading preprocessor: openpose_full
2023-08-20 06:33:17,788 - ControlNet - INFO - preprocessor resolution = 768
100%|████████████████████████████████████████████████████████████| 20/20 [00:05<00:00,  3.57it/s]
Total progress: 100%|████████████████████████████████████████████| 20/20 [00:05<00:00,  3.61it/s]
2023-08-20 06:33:47,067 - ControlNet - INFO - Loading model from cache: control_v11p_sd15_openpose [cab727d4]
2023-08-20 06:33:47,070 - ControlNet - INFO - Loading preprocessor: openpose_full
2023-08-20 06:33:47,070 - ControlNet - INFO - preprocessor resolution = 768
100%|████████████████████████████████████████████████████████████| 50/50 [00:13<00:00,  3.80it/s]
Total progress: 100%|████████████████████████████████████████████| 50/50 [00:13<00:00,  3.72it/s]
2023-08-20 06:34:38,381 - ControlNet - INFO - Loading model from cache: control_v11p_sd15_openpose [cab727d4]
```

### Additional information

According to [this post](https://github.com/lllyasviel/ControlNet/discussions/12), we can directly use the webui to control any SD 1.X models. I assume that also includes finetuned SD models. I wonder what I did wrong. Do I need to retrain ControlNet on my finetuned models?
